rwengine/src/ai/AIGraph.cpp

```diff
@@ -113,7 +114,10 @@ void AIGraph::gatherExternalNodesNear(const glm::vec3& center,
             auto& external = gridNodes[i];
             for (AIGraphNode* node : external) {
```

Comment:
This loop looks like a good candidate for using ``copy_if``.

Reply:
Is it possible to use outer variables for copy_if, because we need to consider the center and the radius?

Reply:
Yes, the lambda function can read external variables. The `&` in `[&](){...`will catch all external variables by reference. But preferably, you'd specify the ones used.

Reply:
Thanks, I hope I got it right.

Reply:
Sure looks like it! 👍